gabrielaEnache/src/main/java/oop/TaskDescription.md

```diff
@@ -0,0 +1,31 @@
+OOP Concepts task:
```

Comment:
Could you please rename the file, because you already have such file in another directory. I would suggest OopTaskDescription.txt

Reply:
Fixed